lib/Doctrine/Common/Cache/RedisCache.php

```diff
@@ -175,10 +175,6 @@ protected function doGetStats()
      */
     protected function getSerializerValue()
     {
-        if (defined('HHVM_VERSION')) {
```

Comment:
This still counts as a BC break. While we are indeed dropping HHVM support, this doesn't mean that we should also break code for them ;-)